Extension/src/platform.ts

```diff
@@ -1,105 +1,105 @@
-                break;
-            case "darwin":
-                version = await PlatformInformation.GetDarwinVersion();
-                break;
-            default:
-                throw new Error(localize("unknown.os.platform", "Unknown OS platform"));
-        }
-
-        return new PlatformInformation(platform, architecture, distribution, version);
-    }
-
-    public static GetArchitecture(): string {
-        const arch: string = os.arch();
-        switch (arch) {
-            case "arm64":
-            case "arm":
-                return arch;
-            case "x32":
-            case "ia32":
-                return "x86";
-            default:
-                return "x64";
-        }
-    }
-
-    private static GetDarwinVersion(): Promise<string> {
-        const DARWIN_SYSTEM_VERSION_PLIST: string = "/System/Library/CoreServices/SystemVersion.plist";
-        let productDarwinVersion: string = "";
-        let errorMessage: string = "";
-
-        if (fs.existsSync(DARWIN_SYSTEM_VERSION_PLIST)) {
-            const systemVersionPListBuffer: Buffer = fs.readFileSync(DARWIN_SYSTEM_VERSION_PLIST);
-            const systemVersionData: any = plist.parse(systemVersionPListBuffer.toString());
-            if (systemVersionData) {
-                productDarwinVersion = systemVersionData.ProductVersion;
-            } else {
-                errorMessage = localize("missing.plist.productversion", "Could not get ProduceVersion from SystemVersion.plist");
-            }
-        } else {
-            errorMessage = localize("missing.darwin.systemversion.file", "Failed to find SystemVersion.plist in {0}.", DARWIN_SYSTEM_VERSION_PLIST);
-        }
-
-        if (errorMessage) {
-            logger.getOutputChannel().appendLine(errorMessage);
-            logger.showOutputChannel();
-        }
-
-        return Promise.resolve(productDarwinVersion);
-    }
-
-    private static GetWindowsVersion(): SupportedWindowsVersions {
-        const version = os.release().split('.');
-        if (version.length > 0) {
-            if (version[0] === '10') {
-                if (version.length > 2 && version[2].startsWith('1')) {
-                    // 10.0.10240 - 10.0.190##
-                    return '10';
-                }
-                // 10.0.22000+
-                return '11';
-            }
-        }
-        return '';
-    }
-}
+/* --------------------------------------------------------------------------------------------
+ * Copyright (c) Microsoft Corporation. All Rights Reserved.
+ * See 'LICENSE' in the project root for license information.
+ * ------------------------------------------------------------------------------------------ */
+
+import * as fs from 'fs';
+import * as os from 'os';
+import * as plist from 'plist';
+import * as nls from 'vscode-nls';
+import { LinuxDistribution } from './linuxDistribution';
+import { getOutputChannelLogger, showOutputChannel } from './logger';

```

Comment:
It's unclear to me why this is changing. It doesn't seem related to the purpose of this PR. Can you revert this file?